scripts/qa_crowbarsetup.sh

```diff
@@ -1233,12 +1241,23 @@ function create_repos_yml()
         SUSE-Enterprise-Storage-2-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/Storage:/2:/x86_64/update/ \
         >> $tmp_yml
 
-    create_repos_yml_for_platform "suse-12.1" "x86_64" "$tftpboot_repos12sp1_dir" \
-        SLES12-SP1-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/12-SP1:/x86_64/update/ \
-        SLE12-SP1-HA-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP1:/x86_64/update/ \
-        SUSE-OpenStack-Cloud-6-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/OpenStack-Cloud:/6:/x86_64/update/ \
-        SUSE-Enterprise-Storage-2.1-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/Storage:/2.1:/x86_64/update/ \
-        >> $tmp_yml
+    if iscloudver 6; then
```

Comment:
@jsuchome as you can see in https://github.com/crowbar/crowbar-core/pull/483 the versions we use for SES are hardcoded for each release. So while it's true we are duplicating code, there is actually no choice for the SES version.

If I kept the block this way is to allow cloud7 to diverge from cloud6, i.e. only support SES12-SP2 and SES30 in cloud7.